common/djangoapps/student/models.py

```diff
@@ -1621,6 +1671,13 @@ def user_can_skip_entrance_exam(cls, user, course_key):
 class LanguageProficiency(models.Model):
     """
     Represents a user's language proficiency.
+
+    Note that we have not found a way to emit analytics change events by using signals directly on this
+    model or on UserProfile. Therefore if you are changing LanguageProficiency values, it is important
+    to go through the accounts API (AccountsView) defined in
+    /edx-platform/openedx/core/djangoapps/user_api/accounts/views.py or the AccountLegacyProfileSerializer
+    in /edx-platform/openedx/core/djangoapps/user_api/accounts/serializers.py so that the events are
```

Comment:
Need to update comment since we weren't able to put the eventing into the serializer.